src/Query/TcpTransportExecutor.php

```diff
@@ -198,16 +198,15 @@ public function query(Query $query)
         $this->writeBuffer .= $queryData;
         if (!$this->writePending) {
             $this->writePending = true;
-            $this->loop->addWriteStream($this->socket, array($this, 'handleWritable'));
+            $this->loop->addWriteStream($this->socket, [$this, 'handleWritable']);
```

Comment:
The `handleWritable` (line 223) can be changed to private and we'll call it through a closure. Might need to update tests, see https://github.com/reactphp/stream/pull/175/files#diff-43369d9ef27744355ce8a83b87ae23a52f63ed9c4d19d041c1d88c3eaca44813R372-R375 on how you could do that

```suggestion
            $this->loop->addWriteStream($this->socket, function () {
                $this->handleWritable();
            });
```

Reply:
See https://github.com/reactphp/stream/pull/175#discussion_r1600439062

Reply:
:+1: